nav2_smac_planner/src/analytic_expansion.cpp

```diff
@@ -360,6 +369,12 @@ typename AnalyticExpansion<Node2D>::NodePtr AnalyticExpansion<Node2D>::tryAnalyt
   const NodeGetter & getter, int & analytic_iterations,
```

Comment:
By Nav2 styling, we typically do `const NodePtr & /*current_node*/` in the method signatures to avoid the `(void)current_node;`'s inline. Can you update for here and other instances? 

I'd also be fine with simply a `const NodePtr &` with no variable as well given these are passthroughs that will never be modified to use them

Reply:
> By Nav2 styling, we typically do `const NodePtr & /*current_node*/` in the method signatures to avoid the `(void)current_node;`'s inline. Can you update for here and other instances?

That's fine.  I should have checked the style elsewhere first.  Will make the change.

Reply:
I'm only really worried about nav2_smac_planner/src/analytic_expansion.cpp & nav2_smac_planner/src/node_hybrid.cpp instances. I'm alot more relaxed on unit tests.

Reply:
Fixed in 4379bbe.